README.md

```diff
@@ -1300,6 +1300,22 @@ References:
 - [Compatibility Standard Definitions Repository](https://github.com/graphql-go/compatibility-standard-definitions)
 - [GraphQL specification repository validation](https://github.com/graphql-go/compatibility-standard-definitions#implementation-details)
 
+##### User Acceptance Definitions
```

Comment:
This title should be updated:

```diff
- ##### User Acceptance Definitions
+ ##### User Acceptance
```

Created a GitHub Issue for it: https://github.com/chris-ramon/thesis-graphql-go/issues/144.